### ISSUE TITLE
[Wii] Imported '.sav' files do not work? 

### ISSUE THREAD

**Issue by GB-CG** [mgba-emu/mgba#169]
I want to move my completed Wario Ware Inc. Save from my VBA-GX to the mGBA games folder, but when launching mGBA, the game does not recognize the completed save. The rom and the save file have matched names. (I'm using Version 0.4-3009-a89bb0b)

**Comment by askotx** [comment]
Savefiles are not 100% compatibles, some of them are, some others not. I could play my save of SMA3, but not from Mario & Luigi Superstar saga

**Comment by endrift** [comment]
That's a bug if they're not 100% compatible, either in VBA-GX or in mGBA, but historically I've been able to import Mario & Luigi saves into mGBA from VBA.

**Comment by endrift** [comment]
@GB-CG if you still have it, can you post the completed save?

**Comment by GB-CG** [comment]
Here: https://www.dropbox.com/s/oszrpl69bgi0mxk/Wario%20Ware%20Inc..sav.zip?dl=0

Sorry for the delay. I was checking my other saves to see if they work. Still checking at it.

**Comment by endrift** [comment]
This doesn't look like a completed save to me, in mGBA or VBA. Might represent a bug in how VBA-GX saves ROMs, but it looks like an empty savegame--there's a name selected (very classy) but the first level hasn't been finished.

**Comment by GB-CG** [comment]
Ok, somehow I've managed to fix it.

What I did was rename the save from 'Wario Ware Inc..sav' to 'Wario Ware.sav' and renamed the rom to 'Wario Ware.gba'. The VBA-GX save file was 'Wario Ware Inc. Auto.sav' originally.

Somehow the extra full stop was the cause of it...? That can't be. I guess I got confused previously when messing with the '.sav' file to make it work on mGBA. Here's the shot of my save now from mGBA. 
![null screen shot 26 11 2015 01 31](https://cloud.githubusercontent.com/assets/11931125/11413141/b23c3c36-93dd-11e5-9268-809df615db00.png)

> there's a name selected (very classy)

Thanks, man. :)
